src/api/lib/tasks/databases.rake

```diff
@@ -83,6 +83,20 @@ namespace :db do
       end
       File.open("#{Rails.root}/db/structure.sql", "w+") { |f| f << new_structure }
     end
+
+    desc "Verify that structure.sql in git is up to date"
+    task verify: :environment do
+      puts "Running rails db:migrate"
+      Rake::Task["db:migrate"].invoke
```

Comment:
I don't see why this would be a problem. And the db:create you propose would also create the tables, right?

I've tried the rake task with a state where #3287 wasn't applied. I also manually changed the strucutre.sql (removed setting the BSTREE indexing), ran the migration and checked the resulting structure.sql.
Running the migrations updates the structure.sql even when there is no migration ins the queue.

I think that is all what we need here, isn't it?